### ISSUE TITLE
publish fails to push conda packages

### ISSUE THREAD

**Issue by prjemian** [BCDA-APS/apstools#346]
```
# (Sat Jul 11 21:25:06 CDT 2020) - - - - - - - - - - - - - - - - - - - - - - upload conda
# (Sat Jul 11 21:25:06 CDT 2020) uploading to anaconda
# (Sat Jul 11 21:25:06 CDT 2020) CHANNEL: aps-anl-dev
Using Anaconda API: https://api.anaconda.org
Using "aps-anl-dev" as upload username
[ERROR] File "/tmp/conda-bld/noarch/apstools-1.3.0rc1-*_0.tar.bz2" does not exist
Using Anaconda API: https://api.anaconda.org
Using "prjemian" as upload username
[ERROR] File "/tmp/conda-bld/noarch/apstools-1.3.0rc1-*_0.tar.bz2" does not exist
# (Sat Jul 11 21:25:08 CDT 2020) Uploaded to anaconda
# (Sat Jul 11 21:15:26 CDT 2020) started publishing script
# (Sat Jul 11 21:25:08 CDT 2020) finished publishing script
(bluesky_2020_5) mintadmin@mint-vm:~/.../projects/apstools$
```

yet this file exists

```
(bluesky_2020_5) mintadmin@mint-vm:~/.../projects/apstools$ ll /tmp/conda-bld/noarch/apstools-1.3.0rc1-*
-rw-rw-r-- 1 mintadmin mintadmin 79K Jul 11 21:19 /tmp/conda-bld/noarch/apstools-1.3.0rc1-py_0.tar.bz2
(bluesky_2020_5) mintadmin@mint-vm:~/.../projects/apstools$ ll /tmp/conda-bld/noarch/apstools-1.3.0rc1-*_0.tar.bz2
-rw-rw-r-- 1 mintadmin mintadmin 79K Jul 11 21:19 /tmp/conda-bld/noarch/apstools-1.3.0rc1-py_0.tar.bz2
```

**Comment by prjemian** [comment]
pushed manually:

```
(bluesky_2020_5) mintadmin@mint-vm:~/.../projects/apstools$ anaconda upload -u aps-anl-dev /tmp/conda-bld/noarch/apstools-1.3.0rc1-py_0.tar.bz2
Using Anaconda API: https://api.anaconda.org
Using "aps-anl-dev" as upload username
Processing '/tmp/conda-bld/noarch/apstools-1.3.0rc1-py_0.tar.bz2'
Detecting file type...
File type is "conda"
Extracting conda package attributes for upload
Creating package "apstools"
Creating release "1.3.0rc1"
Uploading file "aps-anl-dev/apstools/1.3.0rc1/noarch/apstools-1.3.0rc1-py_0.tar.bz2"
 uploaded 82 of 82Kb: 100.00% ETA: 0.0 minutes
Upload complete

conda package located at:
https://anaconda.org/aps-anl-dev/apstools

(bluesky_2020_5) mintadmin@mint-vm:~/.../projects/apstools$ anaconda upload -u prjemian /tmp/conda-bld/noarch/apstools-1.3.0rc1-py_0.tar.bz2 
Using Anaconda API: https://api.anaconda.org
Using "prjemian" as upload username
Processing '/tmp/conda-bld/noarch/apstools-1.3.0rc1-py_0.tar.bz2'
Detecting file type...
File type is "conda"
Extracting conda package attributes for upload
Creating package "apstools"
Creating release "1.3.0rc1"
Uploading file "prjemian/apstools/1.3.0rc1/noarch/apstools-1.3.0rc1-py_0.tar.bz2"
 uploaded 82 of 82Kb: 100.00% ETA: 0.0 minutes
Upload complete

conda package located at:
https://anaconda.org/prjemian/apstools
```

**Comment by prjemian** [comment]
fixed in fc84fa0
